content/en/about/community/join/index.md

```diff
@@ -45,9 +45,16 @@ Once you've done that, you can head straight to our
 {{< /community_item >}}
 
 {{< community_item logo="./group.svg" alt="Working Groups" >}}
-If you'd like to contribute to the Istio project, learn [how we do things](https://github.com/istio/community/blob/master/README.md) and
-consider participating in our
-[working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
+If you'd like to contribute to the Istio project, please read about our
+[code of conduct](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct), our
+[contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md), and
+consider participating in our [working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
+{{< /community_item >}}
+
+{{< community_item logo="./group.svg" alt="Governance" >}}
```

Comment:
Change icon as suggested in https://github.com/istio/istio.io/pull/8382#issuecomment-718821693